subscribie/blueprints/admin/ResetSite.py

```diff
@@ -46,3 +48,24 @@ def remove_transactions():
     msg = {"msg": "all transactions deleted"}
 
     return jsonify(msg)
+
+
+@admin.route("/delete-connect-account", methods=["GET"])
+@login_required
+def delete_connect_account():
+    if os.getenv("FLASK_ENV") != "development":
+        msg = {"msg": "Error. Only possible in development mode"}
+        return jsonify(msg), 403
```

Comment:
Nice error handling , especially the `403` return code :+1: useful for debugging